docs/source/developer/interface/command-line-parameters.md

```diff
@@ -0,0 +1,35 @@
+# Command Line Parameters
+
+You can launch an interface with these parameters.
+
+Displays help information
+
+    -h, --help
+
+Displays version information
+
+    -v, --version
+
+ICE-Server address
+
+    -i <IP:PORT or HOSTNAME:PORT>
+
+Domain Server UUID
+
+    -d <uuid>
+
+Request a temporary domain name
```

Comment:
```suggestion
Requests a temporary domain name
```

Reply:
Kind of a funny thing for the interface to be doing but oh well...